python/rdesigneur/rdesigneur.py

```diff
@@ -28,7 +28,7 @@
 import time
 
 import rdesigneur.rmoogli as rmoogli
-import rdesigneur.fixXreacs as fixXreacs
+import fixXreacs
```

Comment:
For python3, this may not work. Need absolute import e.g. `import moose.fixXreacs` or `from . import fixXreac`  .


Add a test script in `tests/python` folder.

Reply:
This is invalid now.